### ISSUE TITLE
Railcraft rails disassemble into progression breaks

### ISSUE THREAD

**Issue by realthecaffiend** [GTNewHorizons/GT-New-Horizons-Modpack#10369]
### Your GTNH Discord Username

thecaffiend#2253 

### Your Pack Version

2.1.2.3qf  (Some config changes, no recipe changes)

### Your Server

Private server

### Type of Server

Vanilla Forge

### Your Expectation

Using "U" on vibrant alloy to see what could be disassembled to make some vibrant crystals to empower dark steel, I noted that HS transition track from railcraft showed up. We had found some in a railcraft building in a town so I loaded it in, expecting to get some parts and a vibrant plate.

### The Reality

This returned a seemingly random selection of plates: Vibrant, conductive, and red alloy. I realized then that there were multiple recipes for the tracks, and tried some other railcraft track items: presumably related to them using the same item ID, outputs varied wildly. Feeding many items in for testing, I received gears and springs made of tungstensteel, iridium, osmium, etc. These could be baked. or alloy smeltered into ingots far earlier than we should have access (Currently, MV, this was in an LV disassembler however). These items could likely be made out of plain steel and then disassembled for a variety of later game alloys, but I didn't try with any that were hand made, just found (I believe the messy NBT of what was used to make the item was removed, which may be related to this?). HS track and HS wye track seem to share 1465:816, and the wye track is made out of fun materials up to neutronium so I suspect if we tried enough we could get neutronium gears out of it. We did get every other option up to that in my limited testing. 

### Your Proposal

A few options:
Simply blacklist rails/railcraft items from the disassembler (I assume they are only disassemble-able due to having assembler recipes anyhow); I'm not sure if it has a blacklist or this is easy to do
Make the disassembler always "pick" the lowest voltage recipe to make the item: The rails I happened to look at all scaled in eu/t costs as they used higher tier materials to make more, so this would mean players could regress better materials to older, and it's hard to think of how this could be exploitative 
Make the disassembler "pick" the recipe corresponding to the disassemblers tier: This would mean that, like the machines limitation, it could only return tier-appropriate gear from items created or found

TL;DR the disassembler is one of my favorite machines as a player, as a clever player can use it to get extremely limited amounts of materials from loot games (We used it to make an MV extruder in very early LV for better bolts, rods, etc) early, but it comes with a downside when it picks a seemingly-random recipe for items that have multiple routes to them. The random behaviour seems like the main problem here and needs some sort of constraints, IMO. 

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by miozune** [comment]
Fixed by https://github.com/GTNewHorizons/GT5-Unofficial/pull/1044

**Comment by realthecaffiend** [comment]
I searched for "rail" but it looks like the issue that references was private so I didn't see it, haha. Thanks for the work and sorry for the duplicate bug!
